modules/aws/sonar-base-instance/setup.tftpl

```diff
@@ -258,11 +258,17 @@ __EOF__
     fi
 }
 
-STATE_DIR=/imperva
-DATA_DIR=$STATE_DIR/data
-LOGS_DIR=$STATE_DIR/logs
-LOCAL_DIR=$STATE_DIR/local
-APPS_DIR=$STATE_DIR/apps
+if [ -z "${base_directory}" ]; then
+  DATA_DIR=/data
```

Comment:
No need for special handling when base_directory is empty, the code in the 'else' is enough

Reply:
Done